Use area-weighted RMSD and Lagrangian seeding (#271)
mercator-ocean/oceanbench: docs/evaluation-methods.rst

The deviation in kilometers between the two sets of drifting particles computed over the challenger datasets and the GLORYS reanalysis dataset.

> The particles are seeded by sampling ocean grid points without replacement using ``cos(latitude)`` area weights, then simulated in a small square area in the Atlantic joining New York coasts and Nouadhibou coasts.

Reviewer: ```suggestion
The particles are seeded by sampling ocean grid points without replacement using ``cos(latitude)`` area weights, then simulated over the area.
```

The deviation in kilometers between the two sets of drifting particles computed over the challenger datasets and the GLO12 analysis dataset.

> The particles are seeded by sampling ocean grid points without replacement using ``cos(latitude)`` area weights, then simulated in a small square area in the Atlantic joining New York coasts and Nouadhibou coasts.

Reviewer: ```suggestion
The particles are seeded by sampling ocean grid points without replacement using ``cos(latitude)`` area weights, then simulated over the area.
```
